go.mod

```diff
@@ -1,23 +1,62 @@
 module github.com/foundriesio/fioctl
 
-go 1.13
+go 1.17
 
 require (
-	cloud.google.com/go/kms v1.4.0 // indirect
-	cloud.google.com/go/pubsub v1.3.1
+	cloud.google.com/go/pubsub v1.21.0
 	github.com/cheynewallace/tabby v1.1.1
 	github.com/docker/go v1.5.1-1.0.20160303222718-d30aec9fd63c
-	github.com/ethereum/go-ethereum v1.10.15
+	github.com/ethereum/go-ethereum v1.10.17
```

Comment:
i'm sure its safe (it's always been), but you might want to run a "fioctl devices config set ...." and make sure your device decrypts its correctly.

Reply:
Probably I need to reset and run tidy again, maybe with some options...
For some weird reason `go mod tidy` upgraded all packages when I told it to upgrade just Golang.

Reply:
Fixed this